app/format-date.js

```diff
@@ -1,6 +1,6 @@
 const moment = require('moment')
 
-const formatDate = (dateToFormat, currentDateFormat = 'YYYY-MM-DD') => {
+const formatDate = (dateToFormat, currentDateFormat = 'DD/MM/YYYY HH:mm') => {
```

Comment:
Would you be a dear and add a unit test module on this

Reply:
Absolutely!  Apologies, just assumed there were some already.  

The fact that this is a bug probably should have been a clue there wasn't...

Reply:
Bit confused between the tests, does `currentDateFormat` refer to the default or current date?

Reply:
It's the format the actual date is stored in